main.tex

```diff
@@ -369,7 +369,26 @@ \subsection{Distribution of packages}
 %
 % ^.
 
-%\subsection{Support of Astropy}
+\subsection{Support of Astropy}
+
+The \astropy project, as of the v2.0 release, does not receive any direct financial support for
+the development of the \astropypkg.  Development of the \astropypkg is supported
+through contributions by individuals to work on astropy in their own personal
+time, contributions from individuals or groups to work on \astropy as 
+part of a research project, or contributions from institutes that allocate 
+people to work on \astropy. 
+A list of organizations that have contributed to \astropy
+are listed in the Acknowledgements.  
+Different models have been proposed for the support of astropy 
+(ie. \cite{2016arXiv161003159M}), 
+but a sustainable, community-based model has not yet been established.
```

Comment:
It's not clear to me what *community-based model* means here - what about instead saying 'but a long-term plan for sustainability has not yet been established' or something like this? I may be misunderstanding this though.